salt/pythontest/config/vsftpd.conf

```diff
@@ -0,0 +1,45 @@
+ftpd_banner=Welcome to pythontest FTP server
+
+# Run vsftp as a standalone server
+listen=YES
+
+# Allow anonymous FTP. (This is what the test suite uses)
+anonymous_enable=YES
```

Comment:
Sure, I think by default anon_root is whatever the home directory of the ftp user is, which is why I change it in the salt file above.